common/network-common/src/main/java/org/apache/spark/network/util/TransportConf.java

```diff
@@ -111,8 +111,30 @@ public int numConnectionsPerPeer() {
   /** Requested maximum length of the queue of incoming connections. Default is 64. */
   public int backLog() { return conf.getInt(SPARK_NETWORK_IO_BACKLOG_KEY, 64); }
 
-  /** Number of threads used in the server thread pool. Default to 0, which is 2x#cores. */
-  public int serverThreads() { return conf.getInt(SPARK_NETWORK_IO_SERVERTHREADS_KEY, 0); }
+  /**
+   * The configured ratio between number of server threads and number of chunk fetch handler
```

Comment:
ratio to me makes it sound like this is a float/double configuration, you are getting this as an Int.  And the way we define this it either needs to be double/float or back to a percent.  
we have a config for # server threads and this config has to apply to that.

Reply:
The thing this patches attempts to do is to guarantee the # server threads is a multiple of the # chunk fetch handler threads. Using the percent approach, it's tricky to configure things correctly, as discussed in the jira ticket.

The ratio parameter could probably use a better name. However, to make the chunk fetch handler EventLoopGroup truly eliminate the RPC message timeout issue, the condition of # server_threads = # chunk_fetch_handler_threads * int_ratio needs to be satisfied.

This would mean that we cannot directly use the user configured # server threads if it cannot be divided by the int_ratio. That's why the change to serverThreads() is introduced below.

Reply:
I think we need to be more clear on the comment.  I realize you have the jira linked, but I think we should say here that this is the divisor or multiplier (depending on phrasing) and that we change the number server threads to make it a multiple based on this config.  The user may find it weird that they explicitly set server threads = 9 and then we change that to 10 based on this multiple.
So I think the comment needs to be very clear on the results they get.